src/components/TutorialBanner.vue

```diff
@@ -0,0 +1,76 @@
+<script>
+import gql from 'graphql-tag'
+
+export default {
+  props: {
+    pageScroll: {
+      type: Boolean,
+      required: false,
+      default: () => false
+    },
+    tenantSlug: {
+      type: String,
+      required: false,
+      default: () => null
+    }
+  },
+  data() {
+    return {
+      dismissed: false
+    }
+  },
+  computed: {
+    showBanner() {
+      let show = this.flowCount === 0
+      if (!this.dismissed && this.pageScroll) {
+        show = !this.pageScroll
+      } else if (this.dismissed) {
+        show = false
+      }
+      return show
+    }
+  },
+  apollo: {
+    flowCount: {
+      query: gql`
+        query {
+          flow_aggregate {
+            aggregate {
+              count
+            }
+          }
+        }
+      `,
+      update: data => data.flow_aggregate.aggregate.count
```

Comment:
@znicholasbrown Changed to this instead. I'm guessing this is similar to `flow_group_aggregate` but it doesn't count the archived flows?